lib/src/main/kotlin/com/lemonappdev/konsist/core/provider/util/KoFileDeclarationProvider.kt

```diff
@@ -70,14 +70,22 @@ internal object KoFileDeclarationProvider {
                             .map { async { parseKotlinFile(it) } }
                             .toList()
                             .awaitAll()
+                            .filterNotNull()
                     }.also { createKoFilesDeclarationDeferred = it }
                 }
 
             currentDeferred.await()
         }
 
-    private suspend fun parseKotlinFile(it: File): KoFileDeclaration =
```

Comment:
There's no need for this function to be suspending, create a coroutineScope and run an async. This function is already ran in an async in the map above.